earth_enterprise/src/fusion/portableglobe/portablemapbuilder.cpp

```diff
@@ -360,7 +363,19 @@ bool PortableMapBuilder::WriteMapPackets(const std::string& qtpath_str,
                                            uint32 level,
                                            uint32 col,
                                            uint32 row) {
-  if (!KeepNode(qtpath_str)) {
+  /*
```

Comment:
very minor - If this is useful to keep around for future metrics gathering efforts, would it be worth putting this commented out code into an `#ifdef` block?

Reply:
That's a good observation.  I've made that change.  Thanks!